src/Spd.Manager.Shared/Spd.Manager.Shared.csproj

```diff
@@ -6,4 +6,9 @@
     <Nullable>enable</Nullable>
   </PropertyGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\Spd.Resource.Repository\Spd.Resource.Repository.csproj" />
```

Comment:
It feels not right that Manager.Shared needs to reference repository.
You should only needs manager layer shared code,
so use PortalUserServiceCategoryCode instead of Enum
and move PortalUserServiceCategoryCode definition to shared project.
for
public enum ContactAuthorizationTypeCode
{
    [Description("Primary Authorized Business Manager Contact")]
    PrimaryBusinessManager,

    [Description("Authorized Business Manager Contact")]
    BusinessManager
}
move it to shared project and merge it with screening authorized contact code.

Reply:
Agree. Fixed.